repose-aggregator/components/filters/ip-user/src/main/scala/org/openrepose/filters/ipuser/IpUserFilter.scala

```diff
@@ -72,7 +71,7 @@ with UpdateListener[IpUserConfig] {
       servletResponse.asInstanceOf[HttpServletResponse].sendError(500)
     } else {
       logger.trace("IP Classification filter handling request...")
```

Comment:
Also unrelated to your changes, but at some point we should fix the log messages to use the new filter name.